5.5/s2i/bin/run

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# Export the Cgroup limits
+set -e
+export_vars=$(cgroup-limits) ; export $export_vars
+set +e
```

Comment:
You don't have to use set here just for one line. You could do this instead:
```
export_vars=$(cgroup-limits) || exit 1
export $export_vars
```
Alternatively, you could do:
```
export_vars=$(cgroup-limits) && export $export_vars || exit 1
```
which is shorter. I've tested it:
```
14:10 $ false && echo export || echo exit
exit
14:10 $ true && echo export || echo exit
export
```